bson/src/main/org/bson/codecs/ValueCodecProvider.java

```diff
@@ -85,7 +85,7 @@ private void addCodecs() {
         addCodec(new CharacterCodec());
         addCodec(new StringCodec());
         addCodec(new SymbolCodec());
-        addCodec(new UuidCodec());
+        addCodec(new OverridableUuidRepresentationUuidCodec());
```

Comment:
This is so we don't override someone's manually added `UuidCodec`

Reply:
Ack